local/php/assert_test.go

```diff
@@ -25,24 +25,24 @@ import (
 	. "gopkg.in/check.v1"
 )
 
-type ComposerSuite struct{}
+type AssertSuite struct{}
```

Comment:
Let's find another name. Maybe `utils.go` would be better.

Reply:
Suits me fine